Source/Meadow.Core.Samples/IO/SerialListener/MeadowApp.cs

```diff
@@ -15,7 +14,7 @@ public class MeadowApp : App<F7FeatherV2>
         public override Task Initialize()
         {
             // instantiate our serial port
-            classicSerialPort = Device.CreateSerialPort(Device.SerialPortNames.Com4, 9600);
```

Comment:
Can't we get the known serial port names from the device instead of using magic strings? Or are they potentially unknown?

Reply:
they are unknown, which is why this refactor.  F7 is a special case, but Windows/Linux/etc the names are dynamic.  The PlatformOS allows you to get a list of the names